### PR TITLE
Adding EOL to Usage Dictionary

### PR DESCRIPTION
**Pull request by sffrench**
Added EOL to the Usage Dictionary. Issue [#48](https://github.com/StyleGuides/WritingStyleGuide/issues/48)

### CODE REVIEW

File: en-US/E.xml [hunk 1]
```diff
@@ -89,6 +89,19 @@
 			</listitem>
 
 		</varlistentry>
+		<varlistentry id="EOL">
+		 <term>EOL</term>
+			<listitem>
+			 <para>
+				 Initialism for "end of line" (n.) and "end-of-line" (adj.)
```
Comment (daobrien) [inline en-US/E.xml:96]: I'd separate these. It'll make it easier when we start using more automation if word classes are on different lines, and it'll bring it inline with what I've been trying to do so far. e.g.,

"_adj., n._ Correct. Do not use 3D, 3-d, or other variations."

File: en-US/E.xml [hunk 2]
```diff
@@ -89,6 +89,19 @@
...
+			 </para>
+				<para>
+				 Neither the noun nor the adjective should be capitalized unless you are documenting a GUI field, label, or similar element, in which case you should use the same capitalization. Capitalization at the beginning of a sentence is acceptable. The initialism is always uppercase.
```
Comment (daobrien) [inline en-US/E.xml:99]: Can we rephrase this to something like "Always use uppercase for the initialism. Do not capitalize the expansion except at the beginning of a sentence. When documenting GUI objects, use the same capitalization as that shown in the GUI."

Something like that.

**Review by daobrien — CHANGES_REQUESTED** [review]
A couple of nit-picks to look at :)

File: en-US/E.xml [hunk 3]
```diff
@@ -89,6 +89,19 @@
 			</listitem>
 
 		</varlistentry>
+		<varlistentry id="EOL">
```
[the remark this answers is not in the document]
Comment (sffrench) [reply en-US/E.xml:92]: No problem, yes you are right, oversight on my part.

File: en-US/E.xml (hunk 1 above)
Comment (sffrench) [reply en-US/E.xml]: Done

File: en-US/E.xml (hunk 2 above)
Comment (sffrench) [reply en-US/E.xml]: Done

**Comment by sffrench** [comment]
Separated word classes and rephrased

File: en-US/E.xml [hunk 4]
```diff
@@ -89,14 +89,17 @@
 			</listitem>
 
 		</varlistentry>
-		<varlistentry id="EOL">
+		<varlistentry id="eol">
 		 <term>EOL</term>
 			<listitem>
 			 <para>
```
Comment (daobrien) [inline en-US/E.xml:95]: ok, you're going to hate me now... Can you add `<emphasis>` around n. and adj. to be consistent with the rest of the book? I know a few cases remain where this hasn't been updated but I try to fix at every opportunity.

**Review by daobrien — APPROVED** [review]
One minor nit-pick (again) :( but then good to go. Thanks

File: en-US/E.xml (hunk 4 above)
Comment (sffrench) [reply en-US/E.xml:95]: Done :)
